pgcommitfest/commitfest/templates/me.html

```diff
@@ -1,6 +1,11 @@
 {%extends "base.html"%}
 {%load commitfest %}
 {%block contents%}
+ <p>
+  <br/>
+  <b>Status summary: </b>{%for id,title,num in statussummary%}<a href="?status={{id}}">{{title}}</a>: {{num}}. {%endfor%}
+ </p>
```

Comment:
Let's move this below the buttons, so it matches the layout of the commitfest page.